src/styles/nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    flex-direction: row;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item:not(:last-of-type) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $text-accent-color;
+    font-family: Roboto, sans-serif;
+    font-size: $nav-font-size;
+    font-weight: 500;
+    height: $header-height;
```

Comment:
The variable `$header-height` is used here, but it is not defined within this file. Ensure that `$header-height` is defined in a file that is imported into this one, or define it here if it's missing.